content/posts/newsletter-february-2024.md

```diff
@@ -0,0 +1,32 @@
+title: Read the Docs Newsletter - February 2024
+date: Jan 1, 2024
+description: Company updates and new features from the last month, current focus, and upcoming features.
+category: Newsletter
+tags: newsletter, community, build-in-public
+authors: Eric Holscher
+status: published
+
+## News and updates
```

Comment:
Forced redirects are now available for everyone, maybe mention the new limits as well?